pandas/util/decorators.py

```diff
@@ -0,0 +1,8 @@
+import warnings
+
+warnings.warn("pandas.util.decorators is deprecated and will be "
+              "removed in a future version, import "
+              "from pandas.util",
+              DeprecationWarning, stacklevel=3)
+
+from pandas.util import cache_readonly  # noqa
```

Comment:
I don't know of anybody using it, but maybe also Appender and Substition to be sure? (or import *)